components/rmrk/Create/SimpleMint.vue

```diff
@@ -159,13 +159,13 @@
     <NeoField>
       <NeoIcon icon="calculator" />
       <span class="pr-2">{{ $t('mint.estimated') }}</span>
-      <Money :value="estimated" inline data-cy="fee" />
+      <BasicMoney :value="estimated" inline data-cy="fee" />
```

Comment:
why this change?

Reply:
`<Money />` was also BasicMoney: `components/shared/format/BasicMoney.vue`

https://github.com/kodadot/nft-gallery/blob/7442eac23aa04504ae2d7d60a8112d39110d4e64/components/rmrk/Create/SimpleMint.vue#L244

I changed the name to match the filename because I noticed that most of the component names are the same as the filename.